subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaAnnotationProcessingIntegrationTest.groovy

```diff
@@ -267,4 +285,50 @@ class JavaAnnotationProcessingIntegrationTest extends AbstractIntegrationSpec {
         then:
         succeeds "compileJava"
     }
+
+    def "wraps processing in build operation"() {
+        given:
+        buildFile << """
+            dependencies {
+                compileOnly project(":annotation")
+                annotationProcessor project(":processor")
+            }
+        """
+        fixture.delay = 1000
```

Comment:
Sleeping a whole second seems excessive.

Reply:
I removed the sleep and changed the assertion to `>= 0`.